magicctapipe/scripts/magic/tests/params_for_hillas_comparison.py

```diff
@@ -0,0 +1,100 @@
+import math
+mars_params = {
+	"event_id" : "MRawEvtHeader_1.fStereoEvtNumber",
+	"length_M1" : "MHillas_1.fLength", "length_M2" : "MHillas_2.fLength", 
+	"width_M1" : "MHillas_1.fWidth", "width_M2" : "MHillas_2.fWidth", 
+	"size_M1" : "MHillas_1.fSize", "size_M2" : "MHillas_2.fSize", 
+	"slope_M1" : "MHillasTimeFit_1.fP1Grad", "slope_M2" : "MHillasTimeFit_2.fP1Grad", 
+	"delta_M1" : "MHillas_1.fDelta", "delta_M2" : "MHillas_2.fDelta", 
+	"cogx_M1" : "MHillas_1.fMeanX", "cogx_M2" : "MHillas_2.fMeanX", 
+	"cogy_M1" : "MHillas_1.fMeanY", "cogy_M2" : "MHillas_2.fMeanY", 
+	# "hmax" : "MStereoPar.fMaxHeight", 
+	# "corex" : "MStereoPar.fCoreX", 
+	# "corey" : "MStereoPar.fCoreY", 
+	# "az" : "MStereoPar.fDirectionAz", 
+	# "zd" : "MStereoPar.fDirectionZd",
+	# "n_islands" : "MImagePar_1./MImagePar_1.fNumIslands",
+	"leakage_M1" : "MNewImagePar_1./MNewImagePar_1.fLeakage1", "leakage_M2" : "MNewImagePar_2./MNewImagePar_2.fLeakage1"
+}
+mcp_params = {
+	"length_M1" : "length", "length_M2" : "length", 
+	"width_M1" : "width", "width_M2" : "width",
+	"size_M1" : "intensity", "size_M2" : "intensity", 
+	"slope_M1" : "slope", "slope_M2" : "slope", 
+	"delta_M1" : "psi", "delta_M2" : "psi", 
+	"cogx_M1" : "x", "cogx_M2" : "x", 
+	"cogy_M1" : "y", "cogy_M2" : "y", 
+	# "hmax" : "h_max", 
+	# "corex" : "core_x", 
+	# "corey" : "core_y", 
+	# "az" : "az", 
+	# "zd" : "alt",
+	# "n_islands" : "n_islands",
+	"leakage_M1" : "intensity_width_1", "leakage_M2" : "intensity_width_2"
```

Comment:
I guess there is a type with intensity_width_2 should be also intensity_width_1

Reply:
Thanks for noticing. It is fixed now!